src/Hydrators/UserHydrator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace ProjMange\Hydrators;
+
+use ProjMange\Entities\UserEntity;
+
+class UserHydrator
+{
+
+    /**
+     * retrieves user data from db
+     * @param \PDO $db
```

Comment:
think you can remove the \ before PDO here as its not actually being used in the code, its just for description purposes.